src/main/java/com/sourcegraph/cody/agent/CurrentConfigFeatures.java

```diff
@@ -0,0 +1,37 @@
+package com.sourcegraph.cody.agent;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentSkipListMap;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.function.Consumer;
+
+public class CurrentConfigFeatures implements Consumer<ConfigFeatures> {
+
+    public static CurrentConfigFeatures INSTANCE = new CurrentConfigFeatures(new ConfigFeatures(true, false));
```

Comment:
FYI Idiomatic way in IntelliJ to create instance is using project or app level services. 
If you will search for `@Service` you will find some examples.
Difference between static instance like yours, and services, is that lifetime and initialisation order/time of services is managed by IJ. You also get some additional benefits like if you inherit from `Disposable` and override `dispose` method you can ensure proper cleaning in the right moment.
It might not make too big difference in your case, just saying :)

Reply:
Done!